### ISSUE TITLE
if the kernel version I use is 5.15.0,tool is v.2.0,whether I need to do patching to the kernel?

### ISSUE THREAD

**Issue by nxnonxno** [jcsaezal/pmctrack#16]

**Comment by jcsaezal** [comment]
PMCTrack v2.0 was released on late June 2021, and by that time Linux 5.15 was not available yet (v5.15.1 was released on early November). We are working on the next PMCTrack release, and I can tell you that changes in the kernel API introduced in 5.14-5.16 kernels broke the compilation of PMCTrack's kernel module as of PMCTrack v2.0. 

We have addressed that problem in our development version of PMCTrack, and will provide full support for these newer kernel versions in the next public release (late December or early January 2023).  Our goal is that patching becomes unnecessary for kernels v5.9.x and above;  the current development version of PMCTrack does work with unpatched v5.15 kernel.
